src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -60,14 +54,13 @@ public MainPageViewModel ViewModel
 
 		private bool keyReleased = true;
 
-		public readonly OngoingTasksViewModel OngoingTasksViewModel;
+		public readonly OngoingTasksViewModel OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
 
 		public MainPage()
 		{
 			InitializeComponent();
-			DataContext = Ioc.Default.GetRequiredService<MainPageViewModel>();
-			OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
```

Comment:
Can you use dependency injection (building off of @lukeblevins work)?

Reply:
I did define & assign at the same time.

Reply:
Fixed like this in the constructor.

```cs
// Dependency Injection
UserSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
Commands = Ioc.Default.GetRequiredService<ICommandManager>();
WindowContext = Ioc.Default.GetRequiredService<IWindowContext>();
SidebarAdaptiveViewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
ViewModel = Ioc.Default.GetRequiredService<MainPageViewModel>();
OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
```

Reply:
This should suffice until we have a proper navigation service.